CMakeLists.txt

```diff
@@ -207,15 +207,16 @@ endif()
 
 # unless you said you didnt want gdal we try to turn it on, if we cant we tell you
 # we don't need it to build tiles (yet)
-set(GDAL_TARGET "")
-if (ENABLE_GDAL AND (ENABLE_SERVICES OR ENABLE_TOOLS OR ENABLE_NODE_BINDINGS))
-  find_package(GDAL QUIET)
-  if (GDAL_FOUND)
-    set(GDAL_TARGET GDAL::GDAL)
-    add_compile_definitions(ENABLE_GDAL)
-    message(STATUS "Gdal support is enabled")
+set(GTIFF_TARGETS "")
+if (ENABLE_GEOTIFF AND (ENABLE_SERVICES OR ENABLE_TOOLS OR ENABLE_PYTHON_BINDINGS OR ENABLE_NODE_BINDINGS))
+  find_package(TIFF REQUIRED)
+  find_package(GeoTIFF REQUIRED)
+  add_compile_definitions(ENABLE_GEOTIFF)
+  if (GeoTIFF_FOUND AND TIFF_FOUND)
+    set(GTIFF_TARGETS TIFF::TIFF geotiff_library)
```

Comment:
`geotiff_library`  is a target we define in our FindGeoTIFF.cmake. I had to name it this way to be compatible with vcpkg, which is sourced directly from geotiff cmake. 

ubuntu 24.04 doesn't have either a .pc or geotiff-config.cmake, the .pc file is only from mid 2024 and debian packages sadly almost never care about cmake files..